src/frontend/Pages/Home.razor

```diff
@@ -1,7 +1,13 @@
 ﻿@page "/"
+@layout MainLayout
+@using MudBlazor
+@using Frontend.Components
+
+<MudThemeProvider/>
+
+
+<PageTitle>MPM-Smart</PageTitle>
+
```

Comment:
As of now, we don't have a home page so please remove this one and show the default dashboard as start page